cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -232,7 +232,10 @@ public class KubeMasterEnvironment implements MasterEnvironment {
   private long workloadIdentityServiceAccountTokenTTLSeconds;
   private String programCpuMultiplier;
   private String cdapInstallNamespace;
-  private KubeJobCleaner kubeJobCleaner;
+
+  public KubeMasterEnvironment() {
+    tasks = new ArrayList<>();
```

Comment:
In the constructor, usually we use `this.tasks`. Just to be consistent with existing code base.